packages/web/src/scss/components/Toggle/_Toggle.scss

```diff
@@ -12,19 +12,23 @@ $_field-name: 'Toggle';
     justify-content: space-between;
     max-width: theme.$max-width;
     margin-block: form-fields-theme.$inline-field-margin-y;
-    cursor: cursors.$form-fields;
 }
 
 .Toggle--fluid {
     max-width: none;
 }
 
 .Toggle__text {
+    width: 100%;
     margin-right: form-fields-theme.$inline-field-gap;
 }
 
 .Toggle__label {
     @include form-fields-tools.inline-field-label();
+    @include form-fields-tools.stretch();
+
+    position: relative;
+    width: 100%;
```

Comment:
Instead of setting the width here, I'd solve it one level above, that is in the `inline-field-label()` mixin. There is `display: inline-block` which can be replaced with `display: block` (I tested it for `Checkbox` and `Radio` too).

```suggestion
```